blob/testing.go

```diff
@@ -0,0 +1,35 @@
+package blob
```

Comment:
will have cyclic dependencies then. blob package depends on blobtests as it invokes GenerateBlobs(that uses blob.NewBlob inside).

Reply:
I see. The solution would be then to put service with tests into `(blob)service` subpkg

Reply:
ok. I'm fine with it

Reply:
the cyclic dependency will be between `blob/blob_test.go` and `blob/blobtest/testing.go`
I would propose this structure:

```
.
├── blob.go
├── blob_test.go
├── helper.go
├── service
│   ├── helper.go
│   ├── service.go
│   └── service_test.go
└── testing.go
```

Reply:
This restructuring aims to avoid importing testing.T within production code. The structure above still has testing.go in the blob pkg meaning the import of `blob` will import still  `T`.

Reply:
Yes. But I was trying to explain that blob_test and testing depends on each other. We cannot break this dependency unless we want to create a blob interface somewhere

Reply:
The only solutions I see right now:
* GenerateBlobs does not takes T.
	* AVoids the need to restructure anything 
* GenerateBlobs returns apptypes Blob and the tests do the conversion automatically

Reply:
Yes. I think so too